### DISCUSSION TITLE
Proposal: show string/word numbers on search results

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#11770] [Ideas]
I am still looking for numbers, and generating some stats with the api and gitlabci.

I think it would be great to provide the  number of strings and words on the results of the search interface, for example here:

https://hosted.weblate.org/search/tor/community-portal/?q=+added%3A2024-03-06+AND+language%3Aen

In the top we could show '37 strings / 300 words' 

This will help evaluate changes in a quicker and more organic way than elaborating reports, and will be available for translators without admin rights.

**Comment by nijel** [comment]
Makes sense, I've created https://github.com/WeblateOrg/weblate/issues/11853
